SPECS/guava/guava.spec

```diff
@@ -20,7 +20,7 @@ Distribution:   Mariner
 
 Name:           guava
 Version:        25.0
-Release:        5%{?dist}
+Release:        6%{?dist}
 Summary:        Google Core Libraries for Java
```

Comment:
Guava has a low priority CVE against it.  CVE-2020-8908

Reply:
Latest version is 31.1
https://github.com/google/guava/tree/v31.1

Reply:
Tried to upgrade to newer version, requires bringing two more packages to do it. Will in in future PR.